maxim/logger/crewai/client.py

```
@@ -50,10 +50,24 @@ def get_log_level(debug: bool) -> int:
 
 class MaximUsageCallback:
+    """Maxim usage callback.
+
+    This class represents a usage callback.
+    """
+
     def __init__(self, generation_id: str):
+        """Initialize a usage callback."""
         self.generation_id = generation_id
 
     def log_success_event(self, kwargs, response_obj, start_time, end_time):
```

Ruff (0.11.9)

Missing return type annotation for public function `log_success_event`

Add return type annotation: `None`

(ANN201)